Bottom: ca2388b9fec3713cd1c0ee00adb0ce3ac2baf5a0
Top:    dfaa6f16be8175823cd40f22c478488db22b0690
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 19:58:30 +0100

hid/gtk: Don't hide / restore the crosshair unnecessarily

ghid_note_event_location() takes care of that when the crosshair
moves.


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 268988d..800ce06 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -447,12 +447,10 @@ ghid_port_button_press_cb (GtkWidget * drawing_area,
   state = (GdkModifierType) (ev->state);
   mk = ghid_modifier_keys_state (&state);
   ghid_show_crosshair (FALSE);
-  HideCrosshair ();
 
   do_mouse_action(ev->button, mk);
 
   ghid_invalidate_all ();
-  RestoreCrosshair ();
   ghid_window_set_name_label (PCB->Name);
   ghid_set_status_line_label ();
   ghid_show_crosshair (TRUE);
